src/features/profile/components/ProfileBadge.tsx

```diff
@@ -10,11 +11,38 @@ interface ProfileBadgeProps {
   icon?: FunctionComponent<IconInterface>
   callToActionIcon?: FunctionComponent<IconInterface>
   callToActionMessage?: string
+  callToActionLink?: string
   testID?: string
 }
 
+const renderCallToAction = (
+  callToActionMessage: string | undefined,
+  callToActionLink: string | undefined,
+  callToActionIcon: FunctionComponent<IconInterface> | undefined
+) => {
+  if (!callToActionMessage || !callToActionLink) {
```

Comment:
```suggestion
  if (!callToActionMessage || !callToActionLink) return null
  
   return {
   ... 
```

petit nitpick de style